src/app/component/ProductCard/ProductCard.component.js

```diff
@@ -130,19 +132,32 @@ export class ProductCard extends PureComponent {
     }
 
     renderVisualConfigurableOptions() {
-        const { availableVisualOptions } = this.props;
+        const { availableVisualOptions, device } = this.props;
+        if (device.isMobile) {
+            return null;
+        }
 
         return (
             <div block="ProductCard" elem="ConfigurableOptions">
-                { availableVisualOptions.map(({ value, label }) => (
-                    <span
-                      block="ProductCard"
-                      elem="Color"
-                      key={ value }
-                      style={ { backgroundColor: value } }
-                      aria-label={ label }
-                    />
-                )) }
+                { availableVisualOptions.map(({ value, label, type }) => (
+                    type === '1' ? (
+                        <span
+                          block="ProductCard"
+                          elem="Color"
+                          key={ value }
+                          style={ { backgroundColor: value } }
+                          aria-label={ label }
+                        />
+                    ) : (
+                        <span
+                          block="ProductCard"
+                          elem="String"
+                          title={ label }
+                          key={ value }
+                        >
+                                { value }
+                        </span>
+                    ))) }
```

Comment:
`type === '1'` is not semantic enough.
Please define the constant with the meaningful name in the component config file and move type detection and appropriate component render to a separate render function.